### ISSUE TITLE
[Bug Report][3.6.14] v-menus not accessible with screen readers

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#20303]
### Environment
**Vuetify Version:** 3.6.14
**Last working version:** 3.5.1
**Vue Version:** 3.4.27
**Browsers:** Chrome 127.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Using a screen reader, open a v-menu and use the arrow keys to move focus up and down the list of options in the menu

### Expected Behavior
The screen reader acknowledges the moving focus and announces the option it highlights

### Actual Behavior
Nothing is announced

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrtVcGK2zAQ/ZVBPWwClkN3lxZMNrD0XCj0GOegtZWijWwLSTYJRv9eSXFsydlsKbS3PVmj9zTzZp5sb3v0LETatRRlaK1pJTjRdJPXAOuSdVBwotRTjkq85/QIr63SbH/CSpCCYiKbti5z5On2QIcrWrdDZONLOuiw4o3OSKFZR3QjbcIehGyEAjMeH1K86HqKAYqGe76QrCLylKMQtGxWlx61uUIsyAnwfKkLTkZQbeXLTYJX0QAGQZwpHUt0O5hZbqxl74UuHJCA1UWPS/sAF0faALIDPVmq58yQjvCWvoVFHYUisGaa003f+0qpj8AY19ycE7c+osEAonZdODg6OhzYc8OaMdsPImmtYXT9kmW8KRPksnj2/DbMhn97/H9nwHsWvGfCzIZ/YMQtK67MiOyIb+9b5jD3Wjg2nuY/NvIn536yF87qX6F1Nype2fhpXvPjVbrybr2yn1a7ir43NlSFZOJ8gh5FIzWUdE9arqE/ZymJJhkslvC0gcWwZ612U8lgO0nqwYvM4O4bZ8UBvtM7MMn/xOE+ZOyGlVn6hXGtDs2hBNmrdVDpq2pq+8/pc1Qo62m23RmLscr1jSsiAsJ505F6Y5BJ0EP6mN5/tfSH9Ev6+RElWrZ09xudqQCY)

### Other comments
3.5.1 is the last release that this worked, none of the Vuetify releases since have worked

**Comment by johnleider** [comment]
If the bug was introduced in v3.5.2, the only change I'm seeing that may have caused it is this https://github.com/vuetifyjs/vuetify/commit/916c9ef9ee4fd655ffefaa097984a433acbb4c51
